store/slices/userSlice.js

```diff
@@ -0,0 +1,36 @@
+//file contains all the reducer logic and actions specifically for the orders
+//following walkthrough: https://towardsdev.com/setup-and-use-redux-toolkit-with-next-js-begineers-guide-5ebc32eef31e
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+    vote: "not-voted",
+    user: {}
+}
+
+export const userSlice = createSlice({
+    name: 'user',
+    initialState,
+    reducers: {
+        //Actions
```

Comment:
Remove the comments in this file.
Optional: Keep the comments above, but like `// File ...`